spec/poros/food_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Food do
+  let(:apple) do
+    Food.new({
+      gtinUpc: "123456",
+      description: 'keeps the doctor away',
+      brandOwner: 'earth',
+      ingredients: 'seed, sunlight, water'
+      })
+
+  end
+
+  it 'has attributes', :vcr do
```

Comment:
You shouldn't need VCR with a PORO test as its logic doesn't require an external API call. You are passing in the hash and testing that it is creating a ruby object with the right attributes -- that's all you need! No mocking necessary